docs/vereisten/index.md

```diff
@@ -1,69 +1,88 @@
 ---
 title: Vereisten
+summary: Overzicht van eisen voor overheden die algoritmes of AI ontwikkelen of gebruiken.
 hide:
 - navigation
 ---
+# Eisen
+Overzicht van eisen voor overheden die algoritmes of AI-systemen ontwikkelen of gebruiken. Deze eisen zijn gebaseerd op de wetten en regels voor het uitvoeren van wettelijke taken, zoals de Grondwet.
 
-Een kernelement van het Algoritmekader is het beschrijven van de vereisten waar overheidsorganisaties aan moeten voldoen als zij algoritmes en AI (laten) ontwikkelen en gebruiken. 
-Er is tal van wet- en regelgeving van toepassing op overheidsorganisaties als zij hun wettelijke taken uitvoeren. 
-Dat geldt ook als algoritmes en AI worden ingezet ter ondersteuning van de bijbehorende werkprocessen. 
-Hier kan worden gedacht aan de Algemene Wet Bestuursrecht, de Grondwet, de AI-verordening (vanaf inwerkingtreding), Auteurswet, de AVG en meer sectorspecifieke wet- en regelgeving. 
-In het Algoritmekader wordt een overzicht gegeven van de vereisten die hieruit voortkomen en van toepassing zijn op algoritmes en AI. 
-
-!!! warning "Opmerking"
-    Het algoritmekader is nog volop in ontwikkeling. 
-    Dit betekent dat de lijst met vereisten die nu zijn opgenomen in het algoritmekader nog onvolledig is, en dat de vereisten later nog kunnen worden aangepast. 
-    Meer vereisten volgen in een volgende versie van het Algoritmekader. 
-
-## Welke vereisten zijn wanneer van toepassing?
-In dit deel van het Algoritmekader kan worden genavigeerd door de verschillende vereisten. 
-Wat precies van toepassing is, is sterk afhankelijk van de specifieke toepassing. 
-Daarom worden de vereisten gekoppeld aan het type technologie (rekenregel, machine learning of generatieve AI) en aan de risico classificatie (niet-impactvol, impactvol en hoge risico). 
-Dit moet gebruikers helpen om te duiden welke vereisten in welke situaties moeten worden nageleefd. 
-
-??? note "Opmerking"
-    Aan deze functionaliteit wordt nog gewerkt. Meer hierover zal volgen in een volgende versie. 
+## Aantal eisen verschilt per situatie
+Welke eisen gelden voor jouw organisatie, hangt af van:
+- de technologie die je gebruikt: rekenregels, machinelearning of generatieve AI
+- het risiconiveau van je product
+- de [fase in de levenscyclus](https://minbzk.github.io/Algoritmekader/levenscyclus/) van je project
```

Comment:
```suggestion
```

Reply:
de levenscyclus is slechts een suggestie wanneer je er aandacht aan moet besteden, maar het bepaalt niet of een vereiste wel of niet van toepassing is.

Reply:
Ok, ik heb levenscyclus verwijderd uit het rijtje.